spec/requests/api/v1/journeys_controller_index_spec.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V1::JourneysController do
+  let(:response_json) { JSON.parse(response.body) }
+
+  describe 'GET /moves/:move_id/journeys' do
+    let!(:supplier) { create(:supplier) }
+    let(:application) { create(:application, owner: supplier) }
+    let(:access_token) { create(:access_token, application: application).token }
+    let(:headers) { { 'CONTENT_TYPE': content_type, 'Authorization': "Bearer #{access_token}" } }
+    let(:content_type) { ApiController::CONTENT_TYPE }
+
+    let(:locations) { create_list(:location, 2, suppliers: [supplier]) }
+    let!(:move) { create(:move, from_location: locations.first, to_location: locations.last) }
+    let!(:last_journey) { create(:journey, move: move, supplier: supplier, client_timestamp: '2020-05-04T09:00:00Z') }
+    let!(:intermediate_journeys) { create_list(:journey, intermediate_journeys_count, move: move, supplier: supplier, client_timestamp: '2020-05-04T08:30:00Z') }
+    let!(:first_journey) { create(:journey, move: move, supplier: supplier, client_timestamp: '2020-05-04T08:00:00Z') }
+    let!(:other_move_journey) { create(:journey, supplier: supplier) } # another journey for a different move, same supplier
+    let!(:other_supplier_journey) { create(:journey, move: move) } # another journey for a different supplier, same move
+
+    let(:intermediate_journeys_count) { 1 }
+    let(:page) { 1 }
+    let(:per_page) { 20 }
+
+    before do
+      get "/api/v1/moves/#{move.id}/journeys?page=#{page}&per_page=#{per_page}", headers: headers, as: :json
```

Comment:
Do we need specify the pagination here?
I was thinking that we could simplify the test removing it